concentration/Scene/TriangleScene.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  TriangleScene.swift
+//  concentration
+//
+//  Created by Vlad Andrushok on 30.01.2023.
+//
+
+import SpriteKit
+
+class TriangleScene: SKScene {
+    
+    let sizeBall: CGFloat = 30
+    
+    override func didMove(to view: SKView) {
+        
+        self.anchorPoint = .init(x: 0.5, y: 0.5)
+        
+        let circul = SKTexture(imageNamed: "ball")
+        let spriteСircul = SKSpriteNode(texture: circul, size: .init(width: sizeBall, height: sizeBall))
+        spriteСircul.physicsBody = SKPhysicsBody(circleOfRadius: sizeBall / 2)
+        spriteСircul.position = .init(x: 0, y: self.frame.maxY - 100)
+        
+        for _ in 0...50 {
+            let copy = spriteСircul.copy() as! SKSpriteNode
+            self.addChild(copy)
+        }
+
+        
+        self.backgroundColor = .cyan
+        
+        let borderBody =  SKPhysicsBody (edgeLoopFrom: self.frame)
+        borderBody.friction =  0
+        self.physicsBody = borderBody
+        
+        
+        addTriangle()
+    }
+    
+    func setTriangle() -> SKSpriteNode {
+        
+        let triangleTexture = SKTexture(imageNamed: "triangle")
+        let spriteTriangle = SKSpriteNode(texture: triangleTexture, size: .init(width: self.frame.width - (sizeBall * 2), height: 100))
```

Comment:
Make a bit smaller so there is a bit more space then just ball size (maybe putting 4 points spacing on each side)